frontend/modules/issues/issues.redux.ts

```diff
@@ -107,6 +108,10 @@ const updateIssueProps = (issuesMap, issueId, props = {}) => {
 	};
 };
 
+const resetComponentState = (state = INITIAL_STATE) => {
+	return { ...state, componentState: INITIAL_STATE.componentState };
```

Comment:
This clears everything in the state in the component. That means that the filters , showpins  or any criteria changed regarding order are cleared.
If this is what we want is ok, otherwise we should use `setActiveIssue()/setActiveRisk()`  and fix it to accept null issues and deselect the issue/risk.

Reply:
You're right, the previous implementation was too little thought out. This has now been corrected.

Reply:
Lets remove this changes because now is just dead code.